modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/definitions/portlet/action/ImportObjectDefinitionMVCActionCommand.java

```diff
@@ -141,8 +141,18 @@ private void _importObjectDefinition(ActionRequest actionRequest)
 		objectDefinition.setName(ParamUtil.getString(actionRequest, "name"));
 
 		if (FeatureFlagManagerUtil.isEnabled("LPS-148856")) {
-			objectDefinition.setObjectFolderExternalReferenceCode(
-				ObjectFolderConstants.EXTERNAL_REFERENCE_CODE_UNCATEGORIZED);
+			String objectFolderExternalReferenceCode = ParamUtil.getString(
+				actionRequest, "objectFolderExternalReferenceCode");
+
+			if (Validator.isBlank(objectFolderExternalReferenceCode)) {
+				objectDefinition.setObjectFolderExternalReferenceCode(
+					ObjectFolderConstants.
+						EXTERNAL_REFERENCE_CODE_UNCATEGORIZED);
+			}
+			else {
+				objectDefinition.setObjectFolderExternalReferenceCode(
+					objectFolderExternalReferenceCode);
+			}
```

Comment:
Hey @G-Sa 
We don't need to do this verification here because there is a method `_getObjectFolderId()` in `ObjectDefinitionResourceImpl.java` that handles this situation.
I think we just need to do this:
```
objectDefinition.setObjectFolderExternalReferenceCode(
           ParamUtil.getString(actionRequest, "objectFolderExternalReferenceCode"));
```
(Please, see what this method returns: `ParamUtil.getString()`)
If the return of this method is Blank String, the method `_getObjectFolder()` will put the ERC of the Uncategorized Folder (which is 0). 
Please, test if my suggestion works :+1: